server/src/com/cloud/network/security/SecurityGroupManagerImpl.java

```diff
@@ -1207,20 +1183,6 @@ private void cleanupUnfinishedWork() {
         }
     }
 
-    private void processScheduledWork() {
```

Comment:
The usage of this method is commented, should we remove that too or keep this?

Reply:
For me, code commented, dead codes, or code not used should always be deleted.

Reply:
+1

Reply:
Alright, see line 211 in CleanupThread, maybe remove the commented method as well?